server/websocket/receiver.go

```diff
@@ -199,7 +199,7 @@ func (rec *Receiver) fetchOnlyLoop() {
 func (rec *Receiver) fetch() error {
 	fetch := store.FetchRequest{
 		Partition: rec.path.Partition(),
-		MessageC:  make(chan store.MessageAndID, 3),
+		MessageC:  make(chan store.FetchedMessage, 3),
```

Comment:
curious why the capacity of the channel is exactly 3 here - is it useful to be larger ?